lib/content-services/src/lib/content-node-selector/content-node-selector.component.ts

```diff
@@ -89,6 +91,14 @@ export class ContentNodeSelectorComponent {
     }
 
     onShowingSearch(value: boolean) {
-        this.disableUploadButton = value;
+        this.showingSearch = value;
+    }
+
+    onCurrentFolder(currentFolder: Node) {
+        this.hasAllowableOperations = this.contentService.hasAllowableOperations(currentFolder, AllowableOperationsEnum.CREATE);
+    }
+
+    isAllowedToUpload() {
```

Comment:
based on the content of the method, shouldn't this be called is**Not**AllowedToUpload ?

Reply:
Updated. Thanks